File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/java/python/atcoder_ABC124_A.py
Here is the Python equivalent of the Java code:

```python
def solve():
    a, b = map(int, input().split())
    if a == b:
        return a * 2
    elif a > b:
        return a * 2 - 1
    else:
        return b * 2 - 1

print(solve())
```

This Python code does the same thing as the Java code. It reads two integers from the user, then checks if they are equal, greater than or less than each other, and returns the appropriate result. The `map` function is used to convert the input string into a list of integers. The `input()` function is used to read a line of input from the user. The `split()` function is used to split the input string into a list of words, and the `int()` function is used to convert each word into an integer. The `print()` function is used to output the result of the `solve()` function.

File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/python/java/atcoder_ABC124_A.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = 0;
        for (int i = 0; i < 2; i++) {
            c += Math.max(a, b);
            if (a >= b) {
                a -= 1;
            } else {
                b -= 1;
            }
        }
        System.out.println(c);
    }
}
```

This Java code does the same thing as your Python code. It reads two integers from the user, then it performs the same operations as your Python code. The `Math.max` function is used to find the maximum of two numbers, and the `if` statement is used to decrement the smaller number by 1 each time. The result is printed to the console.
